src/operator/subgraph/mkldnn/mkldnn_conv_post_quantize_property.cc

```diff
@@ -107,7 +107,11 @@ class SgMKLDNNConvPostQuantizeProperty : public SubgraphProperty {
     }
   }
   static SubgraphPropertyPtr Create() {
-    return std::make_shared<SgMKLDNNConvPostQuantizeProperty>();
+    auto property = std::make_shared<SgMKLDNNConvPostQuantizeProperty>();
+    property->SetAttr<std::string>("prop_name",
```

Comment:
Nit: can we use "property_name" or just "name" here? Because "prop" also stands for "propagation" somewhere.